netaddr_test.go

```diff
@@ -337,12 +337,12 @@ func TestFromStdIP(t *testing.T) {
 			name: "v6",
 			fn:   FromStdIP,
 			std:  net.ParseIP("::1"),
-			want: IPv6Raw([...]byte{15: 1}),
+			want: IPFrom16([...]byte{15: 1}),
 		},
 		{
 			name: "4in6-unmap",
 			fn:   FromStdIP,
-			std:  net.ParseIP("1.2.3.4"),
+			std:  net.ParseIP("1.2.3.4").To16(),
```

Comment:
this change shouldn't be doing anything, right? I'd prefer you keep the old line alone and add a second test, even if it's currently redundant. It'd be interesting if it one of the cases failed in a future Go release.